selfdrive/test/process_replay/process_replay.py

```diff
@@ -447,6 +538,24 @@ def _replay_single_process(cfg, lr, fingerprint, disable_progress):
       return log_msgs
 
 
+def setup_vision_ipc(cfg, lr):
+  assert len(cfg.vision_pubs) != 0
+
+  device_type = next(msg.initData.deviceType for msg in lr if msg.which() == "initData")
+  is_eon = device_type == "eon"
```

Comment:
we should setup a dict or helper function for this

Reply:
no longer need this